service/src/main/kotlin/io/provenance/api/frameworks/web/external/provenance/ProvenanceHandler.kt

```diff
@@ -120,4 +126,20 @@ class ProvenanceHandler(
             )
         )
     }.foldToServerResponse()
+
+    suspend fun checkCustody(req: ServerRequest): ServerResponse = kotlin.runCatching {
+        getSigner.execute(
+            GetSignerRequest(
+                uuid = req.getUser(),
+                account = AccountInfo()
```

Comment:
should probably pass this in as it contains the `KeyManagementConfig` object that allows configuring your kms

Reply:
updated

Reply:
Are we okay with parsing and acting on a GET request body?

Reply:
Hmm.. I'm torn because it's frowned upon to act on the body of a GET request, but the way we have it is just to optionally override the configuration of the underlying KMS. That config would be no fun to parse via query string parameters. Invokers would either always use it or never use it.